run/GCHP/HEMCO_Config.rc.templates/HEMCO_Config.rc.carbon

```diff
@@ -90,9 +90,7 @@ Mask fractions:              false
     --> AEIC2019_DAILY         :       false    # 2019 (daily data)
     --> AEIC2019_MONMEAN       :       true     # 2019 (monthly-mean data)
     --> AEIC_SCALE_1990_2019   :       true     # Scale to year in 1990-2019
-    --> CEDSv2                 :       true     # 1750-2019
-    --> CEDS_GBDMAPS           :       false    # 1970-2017
-    --> CEDS_GBDMAPS_byFuelType:       false    # 1970-2017
+    --> CEDS_01x01             :       ${RUNDIR_USE_CEDS}    # 1980-2019
```

Comment:
This should be "true" based on previous version

Reply:
Now resolved in 1ef383b65d4a54249b1421d5fa3049cfc7ddb912